src/file/format/yaml.rs

```diff
@@ -5,6 +5,7 @@ use std::mem;
 
 use yaml_rust as yaml;
 
+use crate::error::{ConfigError, Unexpected};
```

Comment:
I'm not sure why you're changing yaml stuff in a PR for json5, please remove this change.